packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -243,6 +250,63 @@ class RenderListWheelViewport
     _hasScrolled();
   }
 
+  /// {@template flutter.rendering.wheelList.curveRate}
+  /// Affect how much the wheel is curved to the left, or the right.
+  ///
+  /// curveRate < 0 means curving to the left, curveRate > 0 means curving to
+  /// the right. The curve depends on the x-coordinate of the center of the
+  /// wheel, where that value is calculated by:
+  /// center.dx = (curveRate + 1) * size.width/2
+  /// (dx = 0 is the left border of the viewport)
+  /// For example, particularly, -1.0 means the center is on the left border
+  /// of the viewport, and 1.0 means it is on the right border.
+  ///
+  /// Defaults to 0.0, which means no curving.
+  /// curveRate should be in the range [-1.0, 1.0].
+  /// {@endtemplate}
+  double get curveRate => _curveRate;
```

Comment:
I'm still having a bit trouble understanding this just from reading without looking at the code. Here's what I'm proposing (let me know if I'm understanding correctly):

name: offAxisFraction
description: 
```
How much the wheel is horizontally off-center, as a fraction of its width.

This property creates the visual effect of looking at a vertical wheel from 
its side where its vanishing points at the edge curves to one side instead 
of looking at the wheel head-on. 

The value is horizontal distance between the wheel's center and the vertical 
vanishing line at the edges of the wheel [I think this is what the math is 
implying], represented as a fraction of the wheel's width. 

The value `0.0` means the wheel is looked at head-on and its vanishing 
line runs through the center of the wheel. Negative values means moving 
the wheel to the left of the observer, thus the edges curve to the right. 
Positive values means moving the wheel to the right of the observer, 
thus the edges curve to the left.

The visual effect causes the wheel's edges to curve rather than moving 
the center. So a value of `0.5` means the edges' vanishing line will touch 
the wheel's size's left edge.
```

Drawing it to physical world equivalents makes it easier to predict its behavior. I think moving the model object in the world instead of moving the world or moving the observer camera makes it easier to quantify numerically. This implies flipping the left/right backwards from how you had it. In terms of naming, we already have things like FractionalOffset. Basing it off of that. Though it also implies the edge is 0.5 width/fraction away from the center.